scipy/stats/_continuous_distns.py

```diff
@@ -4171,6 +4171,121 @@ def _entropy(self, c):
 wrapcauchy = wrapcauchy_gen(a=0.0, b=2*pi, name='wrapcauchy')
 
 
+class gennorm_gen(rv_continuous):
+    """A generalized normal continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `gennorm` is [1]::
+
+                                     beta
+        gennorm.pdf(x, beta) =  ---------------  exp(|x|**beta)
+                                2 gamma(1/beta)
+
+    `gennorm` takes ``beta`` as a shape parameter.
+    For beta = 1, it is identical to a Laplace distribution.
+    For beta = 2, it is identical to a normal distribution.
```

Comment:
Not really: scipy's normal is defined with `exp(-x**2 / 2). Which corresponds to alpha=2 in the Wikipedia definition. Either use alpha=2 or ".. identical to a normal distribution with ``scale=2``." Any choice needs to be supplemented with a test asserting the equivalence in test_distributions.py